### PR TITLE
adding 4.0 promotion tag

### PR DESCRIPTION
**Pull request by shawn-hurley**

### CODE REVIEW

File: ci-operator/jobs/openshift/ansible-service-broker/openshift-ansible-service-broker-master-presubmits.yaml [hunk 1]
```diff
@@ -192,6 +193,278 @@ presubmits:
             cpu: 10m
       serviceAccountName: ci-operator
     trigger: '(?m)^/test (?:.*? )?operator-lint(?: .*?)?$'
+  - agent: kubernetes
```
Comment (petr-muller) [inline ci-operator/jobs/openshift/ansible-service-broker/openshift-ansible-service-broker-master-presubmits.yaml]: All these new jobs look suspicious, we got rid of the `...__rhel.yaml` variant files a while ago. How did these end up here?

Comment (shawn-hurley) [reply ci-operator/jobs/openshift/ansible-service-broker/openshift-ansible-service-broker-master-presubmits.yaml]: I think that was my fault. I had a local file not in the commit when I generated the jobs. 

I removed it and re-generated and nothing changed though.

Comment (petr-muller) [reply ci-operator/jobs/openshift/ansible-service-broker/openshift-ansible-service-broker-master-presubmits.yaml]: Yeah the generator never removes any jobs, even if it generated them before - you'll need to remove the jobs manually.

**Comment by petr-muller** [comment]
I'm investigating the `.../images` rehearsal failure in the meantime, that definitely looks weird.

File: ci-operator/jobs/openshift/ansible-service-broker/openshift-ansible-service-broker-master-presubmits.yaml (hunk 1 above)
Comment (shawn-hurley) [reply ci-operator/jobs/openshift/ansible-service-broker/openshift-ansible-service-broker-master-presubmits.yaml]: I removed it and restarted the job. Thanks for the help!

**Comment by petr-muller** [comment]
@shawn-hurley The rehearsal is actually correct, that job would fail indeed, ci-operator fails to build a `--target=[release:latest]` target if the config does not specify the `tag_specification` stanza, which describes where the "rest of openshift" lives.

**Pull request merged**